### PR TITLE
Fast MathJax rendering

### PR DESCRIPTION
**Pull request by Dantemss**
Gotta render fast!

### CODE REVIEW

**Comment by Dantemss** [comment]
I was bored :)

File: src/helpers/mathjax.coffee [hunk 1]
```diff
@@ -34,10 +34,15 @@ typesetMath = (root) ->
...
-    window.MathJax.Hub.Queue(['Typeset', MathJax.Hub, node])
     # mark node as processed
     node.classList.add('math-rendered')
 
+  # convert NodeList to Array
+  node_array = Array.prototype.slice.call(nodes)
+
+  # submit all nodes at once for optimal rendering performance
+  window.MathJax.Hub.Queue(['Typeset', MathJax.Hub, node_array])
+
```
Comment (nathanstitt) [inline src/helpers/mathjax.coffee]: Wow if this works, it should indeed be more efficient.  I wasn't aware that you could pass an array into `Hub.Queue`

You could just use underscore's _.toArray though:
`window.MathJax.Hub.Queue(['Typeset', MathJax.Hub, _.toArray(nodes)])`

That's assuming that `Hub.Queue` doesn't work with just a  `NodeList` ?

Comment (Fredasaurus) [inline src/helpers/mathjax.coffee]: When Fred gets bored he reads about sports.

Comment (Dantemss) [inline src/helpers/mathjax.coffee]: Feel free to patch it, Nathan, or I can do it tomorrow.

Comment (Dantemss) [inline src/helpers/mathjax.coffee]: Passing a NodeList makes Mathjax blow up.

**Comment by nathanstitt** [comment]
@Dantemss looks good and does seem to render faster.

**Comment by Dantemss** [comment]
Woohoo!

**Comment by Dantemss** [comment]
Keep in mind that this will probably still be slow if some element has `class="math"`, since that uses separate rendering logic for whatever reason.

**Pull request merged**